bites_test.py

```diff
@@ -1,91 +1,112 @@
 import unittest
-from bites import initialise_ants
-from bites import initialise_trail
+from constants import ANTS
+from constants import TOKENS_FOR_TRAIL
+from bites import Bites
 from bites import move_ant
 from bites import take_food
-from bites import initialise_anthill
 from bites import place_ant_on_anthill
 # from bites import
 
+class BitesInitTest(unittest.TestCase):
+  def test___init___method_works(self):
+  # test 36
+  # This test was written rertrospectively; the __init__ method came about 
+  # naturally during refactoring of the code into the Bites class.
+    test_ants = ['purple', 'yellow']
+    test_tokens = {
+      'grapes': 1,
+      'cheese': 1}
+    bites_game = Bites(test_ants, test_tokens)
+    expected_ants = {
+      'purple': None,
+      'yellow': None}
+    expected_trails = [
+      ['grapes', 'cheese'],
+      ['cheese', 'grapes']]
+    expected_anthill = [None, None]
+    self.assertIsInstance(bites_game, Bites)
+    self.assertEqual(bites_game.ants, expected_ants)
+    self.assertIn(bites_game.trail, expected_trails)
+    self.assertEqual(bites_game.anthill, expected_anthill)
+
 class InitialiseAntsTest(unittest.TestCase):
   # test 1
   def test_can_initialise_one_ant(self):
     ants = ["red"]
+    bites_game = Bites(ants, TOKENS_FOR_TRAIL)
```

Comment:
See comment above regarding not using the constants. I realise you need the tokens in order to initialise the Bites class, but since you're not testing the trail here, you could use `tokens_for_trail = {}` an empty dictionary as the tokens and this test should still pass I expect.